cmd/erasure-server-pool-decom.go

```diff
@@ -373,7 +338,61 @@ func (p *poolMeta) load(ctx context.Context, pool *erasureSets, pools []*erasure
 		}
 	}
 
-	return len(rememberedPools) != len(specifiedPools), nil
+	update := len(rememberedPools) != len(specifiedPools)
+	if update {
+		for k, pi := range rememberedPools {
+			if pi.decomstarted && !pi.completed {
+				return false, fmt.Errorf("pool(%s) = %s is being decommissioned, disallows modifying pool command line. Please complete the decommission in progress", humanize.Ordinal(pi.position+1), k)
+			}
+		}
+	}
+	return update, nil
+}
+
+func (p *poolMeta) load(ctx context.Context, pool *erasureSets, pools []*erasureSets, validate bool) (bool, error) {
```

Comment:
What is the returned bool?

Reply:
it tells `Init()` to write a new pool.meta or keep the current one. I can actually change this behavior now.

Reply:
@harshavardhana I meant for you to document it :)

Reply:
Its changed to make it more evident.